doc/source/user_guide/reshaping.rst

```diff
@@ -476,6 +476,14 @@ rows and columns:
 
    df.pivot_table(index=["A", "B"], columns="C", margins=True, aggfunc=np.std)
 
+Additionally, you can call :meth:`DataFrame.stack` to display a pivoted DataFrame
+as having a multi-level index:
+
+.. ipython:: python
+
+    table = df.pivot_table(index=["A", "B"], columns="C", margins=True, aggfunc=np.sum)
```

Comment:
can you just make the assignment on L479 and show table there (and then just `table.stack()` here) rather than repeating